packages/backend-local/package.json

```diff
@@ -1,12 +1,21 @@
 {
   "name": "backend-local",
   "version": "0.0.1",
+  "type": "module",
   "scripts": {
-    "dev": "node index.js"
+    "dev": "nodemon --exec ts-node --esm ./index.ts"
   },
   "dependencies": {
     "cors": "^2.8.5",
     "dotenv": "^16.3.1",
     "express": "^4.18.2"
+  },
+  "devDependencies": {
+    "@types/cors": "^2.8.17",
+    "@types/dotenv": "^8.2.0",
+    "@types/express": "^4.17.21",
+    "@types/node": "^20.10.6",
+    "nodemon": "^3.0.2",
+    "ts-node": "^10.9.2"
```

Comment:
I was receiving this error when executing `yarn backend-local` => _Error: Cannot find module 'typescript'_

Fixed adding `"typescript": "^5.1.6"` dependency here.

Reply:
I thought it's installed on the parent repo and it's enough, but I'm wrong. It seems I installed ts globally. And yes we need it in backend package too, thanks!